syntax.php

```diff
@@ -105,7 +105,7 @@ function handle($match, $state, $pos, Doku_Handler $handler) {
         $data = array('content' => $match);
         foreach (array_keys($this->map) as $rxmatch) {
             if ($match === $rxmatch ||
-                    ($this->map[$rxmatch]['i'] && utf8_strtolower($match) === $rxmatch)) {
+                    ($this->map[$rxmatch]['i'] && strtolower($match) === $rxmatch)) {
```

Comment:
This breaks UTF-8 compatibility

Reply:
Good point. Would you settle for mb_strtolower then? Not perfect either but it will handle most of UTF-8.